content/en/docs/started/workstation/minikf-aws.md

```diff
@@ -0,0 +1,191 @@
+   page).
+
+   ---
+   
+   <img src="/docs/images/minikf-aws/minikf-aws-security-group.png"
+       alt="Create a new security group"
+       class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. Choose an existing **Key Pair** or create a new one. If you create a new one,
+   make sure you click on the **refresh** icon to update the contents of the
+   list, so that your new key pair appears in it.
+   
+   <img src="/docs/images/minikf-aws/minikf-aws-security-settings-key-pair.png"
+       alt="Select a Key Pair or create a new one"
+       class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. Click **Launch**. You will then see a message informing you about the
+   successful installation. The message contains a link to view the instance on
+   the EC2 Console. Click on it.
+
+   <img src="/docs/images/minikf-aws/minikf-aws-successful-deployment.png"
+       alt="Successful deployment message"
+       class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. You should now see a screen like this. Click on the **Instance ID** of the
+   created instance
+
+   <img src="/docs/images/minikf-aws/minikf-aws-ec2-console.png"
+       alt="Instance on EC2 Console"
+       class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. You will then see this page. Click **Connect** to connect to the instance
+   using EC2 Instance Connect
+
+   ---
+   **NOTE**
+
+   This will only work if you have created a security group based on seller
+   settings, as in step 10, or you have configured a security group based on the
+   usage instructions. Otherwise, you need to SSH to the instance manually, see:
+   https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AccessingInstancesLinux.html
+
+   ---
+   
+   <img src="/docs/images/minikf-aws/minikf-aws-connect.png"
+       alt="Connect to the instance using EC2 Instance Connect"
+       class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. The **User name** will be prefilled. Click **Connect**
+
+   <img src="/docs/images/minikf-aws/minikf-aws-ec2-connect.png"
+       alt="EC2 Instance Connect"
+       class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. Follow the on-screen instructions and type **minikf** to view the progress of
+   the deployment
+
+   <img src="/docs/images/minikf-aws/minikf-aws-view-progress.png"
+       alt="View MiniKF deployment progress"
+       class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. Wait for a few minutes
+
+   <img src="/docs/images/minikf-aws/minikf-aws-progress.png"
+       alt="Wait for the installation to complete"
+       class="mt-3 mb-3 p-3 border border-info rounded"> 
+
+1. Once the installation is completed you will see this screen
+
+   <img src="/docs/images/minikf-aws/minikf-aws-provisioning-completed.png"
+       alt="MiniKF installation completed"
+       class="mt-3 mb-3 p-3 border border-info rounded">
+
+1. Follow the URL and log in to MiniKF using the provided username and password.
```

Comment:
```suggestion
1. Follow the URL and log into MiniKF using the provided username and password.
```

Reply:
The spelling was correct here, see this link: https://helpcenter.veeam.com/docs/styleguide/tw/log_in_to.html